### DISCUSSION TITLE
Warning: ob_end_flush(): Session ini settings cannot be changed after headers have already been sent

### DISCUSSION THREAD

**Discussion by yoshiakiokuma** [symfony/symfony#43140] [Q&A]
**Symfony version(s) affected**: 4.4

**Description**
Warning: ob_end_flush(): Session ini settings cannot be changed after headers have already been sent in /usr/local/rms/agency/vendor/symfony/symfony/src/Symfony/Component/HttpFoundation/Response.php on line 1239

**How to reproduce**
php.ini is untouched after installed. Nothing is changed.  We are now seeing this warning on all symfony pages.

**Possible Solution**
no ideas.  We have tried all versions of Component/HttpFoundation without security issues.

**Additional context**

**Comment by GromNaN** [comment]
Can you get the stack trace of the warning ?
This may happen if an other error occurs early in the app run.

**Comment by yoshiakiokuma** [comment]
Hi, So I added debug_print_backtrace(); in line #1233. What else can I do?

   1231     public static function closeOutputBuffers(int $targetLevel, bool $flush): void
   1232     {
   1233         debug_print_backtrace();

Warning: ob_end_flush(): Session ini settings cannot be changed after headers have already been sent in /usr/local/rms/agency/vendor/symfony/symfony/src/Symfony/Component/HttpFoundation/Response.php on line 1241
#0 Symfony\Component\HttpFoundation\Response::closeOutputBuffers(0, 1) called at [/usr/local/rms/agency/vendor/symfony/symfony/src/Symfony/Component/HttpFoundation/Response.php:386] #1 Symfony\Component\HttpFoundation\Response->send() called at [/var/www/html/app.php:31]
